scripts/get-service-editing-urls.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+"""
+Get service editing urls for each page in DOS service flow.
+
+"""
+from dmutils import content_loader
+
+
+def main():
+    lots = ['digital-outcomes', 'digital-specialists', 'user-research-studios',
+            'user-research-participants']
+    loader = content_loader.ContentLoader('../digitalmarketplace-frameworks')
+    loader.load_manifest('digital-outcomes-and-specialists', 'services', 'edit_submission')
+    manifest = loader.get_manifest('digital-outcomes-and-specialists', 'edit_submission')
+
+    url = '/suppliers/frameworks/<framework_slug>/submissions/<lot_slug>/<service_id>'
+    url = '/suppliers/frameworks/{}/submissions/{}/{}'
```

Comment:
Didn't mean to leave the first url line in.  Wasn't trying to do a keyword substitution as there don't seem to be any particular standards with these scripts?  But I can change it to that.